runtime.txt

```diff
@@ -1 +1 @@
-python-3.9.16
+python-3.10.7
```

Comment:
Any reason why we are not on the latest 3.10 patch? 3.10.10 was released on Feb 8th.

Reply:
Miscellaneous bugfixes in the changelog from 3.10.7 to 3.10.10 as far as I can tell.

Reply:
Oops, no reason, just Google lied to me when I searched latest python version 😔